client/state/jetpack-remote-install/test/reducer.js

```diff
@@ -0,0 +1,62 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import reducer, { error, isComplete } from '../reducer';
+import {
+	jetpackRemoteInstall,
+	jetpackRemoteInstallComplete,
+	jetpackRemoteInstallUpdateError,
+} from '../actions';
+
+const url = 'https://yourgroovydomain.com';
+const errorCode = 'INVALID_CREDENTIALS';
+
+describe( 'reducer', () => {
+	test( 'should export expected reducer keys', () => {
+		const state = reducer( undefined, {} );
+
+		expect( state ).toHaveProperty( 'isComplete' );
+		expect( state ).toHaveProperty( 'error' );
+	} );
+
+	describe( 'isComplete', () => {
+		test( 'should store install complete by url', () => {
+			const state = isComplete( undefined, jetpackRemoteInstallComplete( url ) );
```

Comment:
Minor nitpick: I think we used to use explicit action objects in our reducer tests, i.e. 

```js
{
	type: JETPACK_REMOTE_INSTALL_SUCCESS,
	url,
}
```
in this case. The rationale being that by using the `jetpackRemoteInstallComplete()` action here, it's not strictly a unit test anymore, but implicitly relies on that action to do what we expect it to do.

I'm not feeling too strongly about this, since these actions are simple enough functions that generate these objects, and it's arguable that the correct behavior is guaranteed since they have their own unit tests.

Reply:
Yes, not much in it, but I agree. Updated.